src/pages/soknad/[uuid]/pdf-netto.tsx

```diff
@@ -41,27 +51,43 @@ export async function getServerSideProps(
   }
 
   let errorCode = null;
+  let personalia = null;
   let soknadState = null;
+  let dokumentkrav = null;
 
   const onBehalfOfToken = await session.apiToken(audienceDPSoknad);
+  const personaliaResponse = await getPersonalia(onBehalfOfToken);
   const soknadStateResponse = await getSoknadState(uuid, onBehalfOfToken);
+  const dokumentkravResponse = await getDokumentkrav(uuid, onBehalfOfToken);
 
   if (!soknadStateResponse.ok) {
     errorCode = soknadStateResponse.status;
   } else {
     soknadState = await soknadStateResponse.json();
   }
 
+  if (dokumentkravResponse.ok) {
+    dokumentkrav = await dokumentkravResponse.json();
+  } else {
+    errorCode = dokumentkravResponse.status;
+  }
+
+  if (personaliaResponse.ok) {
+    personalia = await personaliaResponse.json();
+  }
+
   return {
     props: {
+      personalia,
       soknadState,
+      dokumentkrav,
       errorCode,
     },
   };
 }
 
-export default function SoknadPage(props: IProps) {
-  if (props.errorCode || !props.soknadState) {
+export default function PdfNettoPage(props: IProps) {
+  if (props.errorCode || !props.soknadState || !props.personalia) {
```

Comment:
Personalia er nice to have i søknaden. Selv om informasjonen om personalia er nede feiler ikke søknad og innsending. Jeg antar det samme skal skje her? Altså at det ikke skal feile så hardt hvis personalia er nede.

Reply:
Det er must have i PDF fordi den skal journalføres og da trenger vi personalia